BrowserKit/Sources/WebEngine/EngineConstants.swift

```diff
@@ -4,8 +4,12 @@
 
 import Foundation
 
-struct EngineConstants {
+/// The constants related to the Web Engine
+public struct EngineConstants {
     static let aboutBlank = "about:blank"
+    static var webserverPort = 6571
```

Comment:
just FYI: We have a bug in UX fundamentals that reader mode fails if the user opens more than one instance of Firefox for instance release and testflight builds because we use the same port, once we made a decision I will update you to have the change in WebEngine if necessary